python/dgl/data/dgl_dataset.py

```diff
@@ -49,6 +49,8 @@ class DGLDataset(object):
         Whether to reload the dataset. Default: False
     verbose : bool
         Whether to print out progress information
+    reverse_edge: bool
```

Comment:
This option is only valid for citation graphs, we won't add it to the base class.